### ISSUE TITLE
Consider Device / Container Size

### ISSUE THREAD

**Issue by tschug** [SFDO-Community-Sprints/Salesforce-Indicators#87]
We may want to consider using the `@api flexipageRegionWidth` and SLDS Layout concept to make the component responsive to its container / device size.

See Resources:
* https://developer.salesforce.com/docs/component-library/documentation/en/lwc/use_width_aware
* https://developer.salesforce.com/docs/component-library/bundle/lightning-layout-item/example
* https://developer.salesforce.com/docs/component-library/bundle/lightning-layout/example
* https://www.lightningdesignsystem.com/utilities/layout/
* https://www.lightningdesignsystem.com/utilities/sizing/

**Comment by JodieM** [comment]
Yes, this will be great. It may be an option on the component to have it display as a Card (with heading / Icon) or a Layout (plain). This fits in with #27 and #73.

**Comment by JodieM** [comment]
See work done in PR #172 as a starting point for this one.

**Comment by JodieM** [comment]
This is still a nice to have.
